kuma/banners/jinja2/banners/banners.html

```diff
@@ -0,0 +1,52 @@
+{% set banners = get_banners() %}
+<div
+    id="developer_needs"
+    class="mdn-cta-container hidden"
+    aria-expanded="false"
+    aria-hidden="true"
+    data-banner="developer_needs"
+>
```

Comment:
Why display this container div at all if there are no banners?
I mean, why not do this:

```jinja
{% if banners %}
  <div id="developer_needs">
    {% for banner in banners %}
    {% endfor %}
    <button/>
  </div>
{% endif %}
```

Reply:
@peterbe That container is going away in the next PR. I need to rewrite some other bits as well and did not want to add additional noise to PR.